src/angular/ngVueLinker.js

```diff
@@ -15,21 +15,26 @@ export function ngVueLinker (componentName, jqElement, elAttributes, scope, $inj
 
   const inQuirkMode = $ngVue ? $ngVue.inQuirkMode() : false
   const vueHooks = $ngVue ? $ngVue.getVueHooks() : {}
+  const vuexStore = $ngVue ? $ngVue.getVuexStore() : null
 
   const watchOptions = {
     depth: elAttributes.watchDepth,
     quirk: inQuirkMode
   }
   watchPropExprs(dataExprsMap, reactiveData, watchOptions, scope)
 
-  const vueInstance = new Vue({
+  const vueOptions = {
     el: jqElement[0],
     data: reactiveData,
     render (h) {
       return <Component {...{ directives }} {...{ props: reactiveData._v }} />
     },
     ...vueHooks
-  })
+  }
+  if (vuexStore) {
```

Comment:
oh, if `Vue.options` is exposed, we can remove the vue hooks to make the plugin provider clearer and simpler :-)